data_prep/graph_dataset.py

```diff
@@ -152,6 +154,10 @@ def _preprocess(self, lower_threshold=4, upper_threshold=50):
             tokenized_text (List): List of tokenized documents texts.
             tokens (List): List of all tokens.
         """
+        if len(self._raw_texts) == 50000:
```

Comment:
Another idea would be do check for the type of `corpus` (or the name) and do do this for specific data classes. Might be a bit nicer because it is less implicit for which data this is done. Also a comment would be nice here, why we are doing it and for which dataset this applies. (Unless you don't want Phillip to read this comment :D)